core/src/main/java/tc/oc/pgm/killreward/KillRewardModule.java

```diff
@@ -62,7 +65,9 @@ public KillRewardModule parse(MapFactory factory, Logger logger, Document doc)
               0)) {
         ImmutableList.Builder<ItemStack> items = ImmutableList.builder();
         for (Element itemEl : elKillReward.getChildren("item")) {
-          items.add(factory.getKits().parseItem(itemEl, false));
+          final ItemStack itemStack = factory.getKits().parseItem(itemEl, false);
+          itemModifier.ifPresent(imm -> imm.applyRules(itemStack));
```

Comment:
If the goal of `ItemModifyModule` is to ensure that _all_ `ItemStack`s are modified to certain rules, I would think that `parseItem` should return the already modified item? The problem here is that the caller has to remember to call `applyRules`, which would be quite easy to forget.

Reply:
Original pgm did it like this and i don't know why, but something tells me it's better not to.

I feel there may be use-cases for parseItem that don't require itemmodding